symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/stream/StreamService.java

```diff
@@ -75,23 +84,59 @@ public OboStreamService obo(AuthSession oboSession) {
   /**
    * {@inheritDoc}
    */
-  public V2StreamAttributes getStreamInfo(String streamId) {
+  @Override
+  public V2StreamAttributes getStreamInfo(@Nonnull String streamId) {
     return executeAndRetry("getStreamInfo",
         () -> streamsApi.v2StreamsSidInfoGet(streamId, authSession.getSessionToken()));
   }
 
   /**
    * {@inheritDoc}
    */
-  public List<StreamAttributes> listStreams(StreamFilter filter) {
+  @Override
+  public List<StreamAttributes> listStreams(@Nullable StreamFilter filter) {
     return executeAndRetry("listStreams",
         () -> streamsApi.v1StreamsListPost(authSession.getSessionToken(), null, null, filter));
   }
 
   /**
    * {@inheritDoc}
    */
-  public void addMemberToRoom( Long userId, String roomId) {
+  @Override
+  public List<StreamAttributes> listStreams(@Nullable StreamFilter filter, @Nullable PaginationAttribute pagination) {
```

Comment:
Please remove `@Nullable` on `PaginationAttribute pagination` and assume it is not null.

Reply:
I think this one should be `@Nullable`. Imagine that in some case, users retrieve the pagination attribute from another method, and they cannot ensure that the attribute is not null, then they have to make their code more complex by adding a null check. :/